lib/cloud/aws/labels.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package aws
+
+// Labels to add to Teleport Resources when importing them from AWS
+const (
+	labelAccountID = "account-id"
+	labelRegion    = "region"
```

Comment:
We have the same labels defined in https://github.com/gravitational/teleport/blob/master/api/types/constants.go#L691-L693, can we reuse those?

Reply:
Yes :+1: